projects/packages/forms/tests/php/contact-form/test-class.contact-form.php

```diff
@@ -1128,18 +1139,22 @@ public function assertValidFieldMultiField( $html, $attributes ) {
 				$this->assertEquals( $option->nodeValue, $attributes['options'][ $i ], 'Input does not match the option' );
 			}
 		} else {
+			$label = $this->getFirstElement( $wrapper_div, 'legend' );
+
+			$this->assertFieldLabel( $wrapper_div, $attributes, 'legend' );
 			$this->assertEquals( 'grunion-field-label', $label->getAttribute( 'class' ), 'label class doesn\'t match' );
 			// Radio and Checkboxes.
 			$labels = $wrapper_div->getElementsByTagName( 'label' );
-			$n      = $labels->length - 1;
+			$n      = $labels->length;
```

Comment:
Since the root `label` is now a `legend`, the number of `label`s in the field now matches the number of options.